Week2/trigger alert.js

```diff
@@ -0,0 +1,15 @@
+delimiter |
+
+CREATE TRIGGER language_row_alert BEFORE INSERT	ON countrylanguage 
+  FOR EACH ROW 
+  BEGIN
+    DECLARE msg VARCHAR(128);
```

Comment:
The idea is correct. However, there is a small typo and therefore I the alert is never triggered, if you correct this, I'll approve your homework.

Reply:
I fixed the typo issue and checked if it works.